Switch Web Share Permissions Policy to *

marcoscaceres opened the pull request:
#### 7de22008cc88d76fccc0d018ef1ffa95c35081d6
<pre>
Switch Web Share Permissions Policy to *
<a href="https://bugs.webkit.org/show_bug.cgi?id=241363">https://bugs.webkit.org/show_bug.cgi?id=241363</a>

Patch by Marcos Cáceres &lt;marcos@marcosc.com &gt; on 2022-06-12
Reviewed by Youenn Fablet and Tim Horton.

* LayoutTests/http/tests/webshare/webshare-allow-attribute-canShare.https-expected.txt:
* LayoutTests/http/tests/webshare/webshare-allow-attribute-canShare.https.html:
* LayoutTests/http/tests/webshare/webshare-allow-attribute-share.https-expected.txt:
* LayoutTests/http/tests/webshare/webshare-allow-attribute-share.https.html:
* Source/WebCore/html/FeaturePolicy.cpp:
(WebCore::FeaturePolicy::parse):

Canonical link: <a href="https://commits.webkit.org/251487@main">https://commits.webkit.org/251487@main</a>
git-svn-id: <a href="https://svn.webkit.org/repository/webkit/trunk@295482">https://svn.webkit.org/repository/webkit/trunk@295482</a> 268f45cc-cd09-0410-ab3c-d52691b4dbfc
</pre>

youennf reviewed (approved):
The commit message should be formatted appropriately:
Title
link to bugzilla
List of changed files.

marcoscaceres committed:
Switch Web Share Permissions Policy to *
https://bugs.webkit.org/show_bug.cgi?id=241363

Patch by Marcos Cáceres <marcos@marcosc.com> on 2022-06-12
Reviewed by Youenn Fablet and Tim Horton.

* LayoutTests/http/tests/webshare/webshare-allow-attribute-canShare.https-expected.txt:
* LayoutTests/http/tests/webshare/webshare-allow-attribute-canShare.https.html:
* LayoutTests/http/tests/webshare/webshare-allow-attribute-share.https-expected.txt:
* LayoutTests/http/tests/webshare/webshare-allow-attribute-share.https.html:
* Source/WebCore/html/FeaturePolicy.cpp:
(WebCore::FeaturePolicy::parse):

Canonical link: https://commits.webkit.org/251487@main
git-svn-id: https://svn.webkit.org/repository/webkit/trunk@295482 268f45cc-cd09-0410-ab3c-d52691b4dbfc
